mix.exs

```diff
@@ -85,7 +86,8 @@ defmodule FunWithFlags.Mixfile do
       {:"test.phx", [&run_tests__redis_pers__phoenix_pubsub/1]},
       {:"test.ecto.postgres", [&run_tests__ecto_pers_postgres__phoenix_pubsub/1]},
       {:"test.ecto.mysql", [&run_tests__ecto_pers_mysql__phoenix_pubsub/1]},
-      {:"test.redis", [&run_tests__redis_pers__redis_pubsub/1]},
+      {:"test.ecto.sqlite3", [&run_tests__ecto_pers_sqlite3__phoenix_pubsub/1]},
+      {:"test.redis", [&run_tests__redis_pers__redis_pubsub/1]}
```

Comment:
Let's keep the trailing comma. GitHub doesn't support clean suggestions in this case, but basically:

```suggestion
      {:"test.redis", [&run_tests__redis_pers__redis_pubsub/1]},
```